godel_process_planning/src/common_utils.cpp

```diff
@@ -91,6 +101,21 @@ bool godel_process_planning::descartesSolve(const godel_process_planning::Descar
   return true;
 }
 
+
+static double calcDefaultTime(const std::vector<double>& a, const std::vector<double>& b,
+                              double max_joint_vel)
+{
+  ROS_ASSERT(a.size() == b.size());
+  ROS_ASSERT(a.size() > 0);
```

Comment:
It's a debug and sanity check tool. If these are tripped then there's a fundamental error in our robot model or how I'm using this function. There's nothing safety critical going on, so killing the program with an error message doesn't seem unreasonable to me.